tests/workflow_tests/base_test.py

```diff
@@ -0,0 +1,46 @@
+#########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+import tempfile
+import os
+from testenv import TestCase
+
+from wagon.wagon import Wagon
+
+from manager_rest import manager_exceptions
+
+
+class WorkflowBaseTest(TestCase):
+    def upload_plugin(self, package_name, package_version):
+        temp_file_path = self._create_wheel(package_name, package_version)
+        response = self.client.plugins.upload(temp_file_path)
+        os.remove(temp_file_path)
+        return response
+
+    def _create_wheel(self, package_name, package_version):
+        module_src = '{0}=={1}'.format(package_name, package_version)
+        wagon_client = Wagon(module_src)
+        return wagon_client.create(
+            archive_destination_dir=tempfile.gettempdir(),
+            force=True)
+
```

Comment:
this is used once and also should be validated in the unit-test and not in the integration test